src/main/java/org/wildfly/security/auth/spi/Attributes.java

```diff
@@ -163,6 +163,15 @@ public void clear() {
     void clear();
 
     /**
+     * Get the collection of values for the given key.  The result may implement {@link SetEntry} if the values
+     * are distinct (for example, a role or group set).
+     *
+     * @param key the attribute name
+     * @return the (possibly empty) attribute value collection
+     */
+    Entry get(String key);
```

Comment:
Just changing the position of this method to get it closed to others defined by this interface.